src/app/shared/pages-scroll-cards/pages-scroll-cards.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'pages-scroll-cards',
+  templateUrl: './pages-scroll-cards.component.html',
+  styleUrls: ['./pages-scroll-cards.component.css']
+})
+
+export class PagesScrollCardsComponent {
+  @Input() requestData: any;
+  @Input() showMeta: boolean;
+
+  getAuthor(page) {
```

Comment:
This is a great option for now and for the initial release of this project. We could look into the possibility of fetching a thumbnail of the person's profile photo later.

Reply:
Awesome!